src/midifile.c

```diff
@@ -456,7 +457,7 @@ static boolean ReadTrack(midi_track_t *track, FILE *stream)
     {
         // Resize the track slightly larger to hold another event:
 
-        new_events = realloc(track->events, 
+        new_events = I_Realloc(track->events, 
```

Comment:
I still think changing this one is a bad idea.

Reply:
Do we also want to introduce safe variants for critical functions like `system()` or `fread()`, which we currently use without checking the return value?